### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by aftabgit786**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+file = open('Murree_weather_2004_Aug.txt', 'r')
+
+content = file.read()
+
```
Comment (Hamma111) [inline main.py]: Always close fielestream: `file.close()`
more info https://realpython.com/read-write-files-python/#opening-and-closing-a-file-in-python

File: main.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+file = open('Murree_weather_2004_Aug.txt', 'r')
```
Comment (Hamma111) [inline main.py]: this only opens one file? we have like hundreds of them, right? where is the rest of code

File: key.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC7kZtvvs8wu4bqxFqLWQOFPgZsouhfNL1W9ZUUAq7qyZwbcp2PnGEUP4GI1qrHzlBEhujNaIEq/m9UAA7fIXIuN2Oq2QteyDeXN8uU3fPaBq9z1m4HVr09XKhaF0ZXn2JkhIp1mpyTwu72QLhw4P7/Ph/XAp5mIU+W1Sy95Jv+y6xX7mmEqgBy1NRb2v6fIsddo1h+ZhnkJfMMBOtMqeZ6oXxOzP328Ne2ssXQCD223KYdiaRcnlOaH4NeRN7wFOMV2cxE4LrNHZJ1/bwZ48m2w7yjI3YWRWgY/QdUcgRFIrG/fj1MG56R9nVq74+rDFDDlM4Oiyas8YVBuz2zuzkzE8ZboabfimnuEEGN3A0/Eziq5s67O7R84q7L7ERhbSPdmBS6uHSkrMhTX3vEitH3v2QS9SlmKxZHyP0CPkgVt7Io9NMjFdmqS5SxVROjw2gkDw1/Qlw3RKWptFItjiTsOBpKo/QXxA4Kpu2WOToGch9F4nMed+L1HkAW7jtxPTE= hammad@Hammads-MacBook-Pro.local
```
Comment (Hamma111) [inline key.txt]: add this in gitignore

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (Hamma111) [inline main.py]: Next time you send me PR, it should have more work done, and not just 6 lines of code

File: main.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+file_values = read_files()
+
+temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        temperatures[file_value[0]] = float(file_value[1])
+
+min_temp_year = min(temperatures, key=temperatures.get)
+max_temp_year = max(temperatures, key=temperatures.get)
```
Comment (Hamma111) [inline main.py]: How is this actually working. what is `key` arg here?

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: main.py (hunk 4 above)
Comment (aftabgit786) [reply main.py]: key= lgany se pehly programe run ho raha tha lekin wo ye kar raha tha k wo file me gya or usny dekha temperature ko pehly usny first line me dekha temp 20 he next line me tmep 21 he to progrmame wahi pe ruk gya agy nai ja raha tha key= lgany k baad wo sari files me  ja k har 1 line ko read kar raha he or jo min() function lgaya he wo minimum number find karny k lya lgaya he ab wo minimum ko find kary ga chahy hmary pass 100000 file hi q na ho sab ja k sab se kam ko nikal k laye ga


next line me max() or key same yahi kaam he lekin yaha pe wo maximum ko nikal k laye ga

Comment (Hamma111) [reply main.py]: good, now do it without `min` or `max` function @aftabgit786

Comment (aftabgit786) [reply main.py]: ok

File: utils/__pycache__/content.cpython-310.pyc
Comment (Hamma111) [inline utils/__pycache__/content.cpython-310.pyc:1]: Exclude the `__pychache__` and `*.pyc` files in your repo. 

Delete the existing __pychache__ folder for it to take effect

File: tasks/t1.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from weatherman.utils.reader import read_files
```
Comment (Hamma111) [inline tasks/t1.py:1]: good job getting it done this way

File: tasks/t1.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+file_values = read_files()
+
+temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        temperatures[file_value[0]] = float(file_value[1])
+
+min_temp_year = min(temperatures, key=temperatures.get)
```
Comment (Hamma111) [inline tasks/t1.py:10]: DO THIS WITHOUT MIN AND MAX FUNCTIONS!!!

File: utils/content.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+def get_contents_of_one_file(file_content):
```
Comment (Hamma111) [inline utils/content.py:1]: you can move this file to `reader.py` as well

File: tasks/t1.py (hunk 6 above)
Comment (Hamma111) [reply tasks/t1.py:10]: do many mistakes. but dont repeat the same mistake again.

**Review by Hamma111 — CHANGES_REQUESTED** [review]

**Pull request merged**